packages/cli/src/commands/fetch.ts

```diff
@@ -216,6 +219,21 @@ export const fetchCommand = async (
   return CliExitCode.AppError
 }
 
+const shouldRecommendAlignMode = async (
+  workspace: Workspace,
+  stateRecencies: StateRecency[],
+  inputServices?: string[],
+): Promise<boolean> => {
+  const newlyAddedServices = stateRecencies
+    .filter(recency => recency.status === 'Nonexistent')
+    .filter(recency => (
+      !Array.isArray(inputServices)
+      || inputServices.includes(recency.serviceName)
+    ))
```

Comment:
style nit:
```suggestion
    .filter(recency => (
      inputServices === undefined
      || inputServices.includes(recency.serviceName)
    ))
```
The reason I think this is more readable is because the type already makes sure this is either `undefined` or an array, generally I think checking for a negative is harder to understand than checking for a positive and specifically checking for `undefined` is a repeating pattern for optional parameters which makes it easier to quickly understand (for me at least) whereas checking whether something is an array usually has a different meaning